server/routes.js

```diff
@@ -292,6 +294,7 @@ exports.isomorphicRoutes = function isomorphicRoutes(
     getClient = require("./api-client").getClient,
     renderServiceWorker = renderServiceWorkerFn,
     publisherConfig = require("./publisher-config"),
+    redirectUrls,
```

Comment:
What is the default value for this?

Reply:
redirectUrls will have two values either it will be an object of an array or a function. taking array is the default value if nothing is passing here.